cfp/index.html

```diff
@@ -0,0 +1,16 @@
+---
+layout: default
+---
+<article id="home">
+  <dl>
+    {% assign now = site.time | date: '%s' | plus: 0 %}
+    {% assign conferences = site.data.conferences | sort: 'start_date' %}
```

Comment:
We probably want to sort by `cfp_date` here, so we get the ones that close first at the top
```suggestion
    {% assign conferences = site.data.conferences | sort: 'cfp_date' %}
```

Reply:
Only issue is that'll push "not yet open" CFPs above "open" CFPs.

Reply:
Hmm right. I guess the "event" of the CFP opening could also be interesting in this view.

Unless we remodel the way we handle the dates, like a `cfp_open_date` and `cfp_close_date` so we also don't need to manually update the `cfp_date` and `cfp_phrase` once the CFP actually opens.

Reply:
In a follow up? Would be good also to refactor all that past data at the same time!

Reply:
Sounds good 👍🏼